geometry_msgs/Pose sawyer_cog
geometry_msgs/Pose sawyer_init